app/forms/state_file/id_disability_form.rb

```diff
@@ -7,6 +7,24 @@ class IdDisabilityForm < QuestionsForm
     validates :primary_disabled, inclusion: { in: %w[yes no], message: :blank }, if: -> { should_check_primary_disabled? }
     validates :spouse_disabled, inclusion: { in: %w[yes no], message: :blank }, if: -> { should_check_spouse_disabled? }
 
+    def initialize(intake = nil, params = {})
+      super
+      already_answered_disability = !intake.primary_disabled_unfilled? && !intake.spouse_disabled_unfilled?
+      if intake.show_mfj_disability_options? && already_answered_disability
+        mfj_disability =
+          if intake.primary_disabled_yes? && intake.spouse_disabled_yes?
+            "both"
+          elsif intake.primary_disabled_yes?
+            "primary"
+          elsif intake.spouse_disabled_yes?
+            "spouse"
+          else
+            "none"
+          end
+        self.mfj_disability ||= mfj_disability
```

Comment:
i had to put the `||` here b/c without it -- on `update` action (and subsequent save) the previous state was saved (b/c the `mfj_disability` is derived from the persisted intake attributes, rather than what the user selected) 

`self.mfj_disability` seems to get set via form attributes when the user makes a selection on the page. We don't want override it if thats present. When the page first loads (`#edit` this is `nil`)

Reply:
open to other approaches, as this is probably a often unused approach